zcncore/get_data.go

```diff
@@ -189,18 +193,21 @@ func GetMintNonce() ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+	var res []byte
 
-	return client.MakeSCRestAPICall(ZCNSCSmartContractAddress, GET_MINT_NONCE, Params{
+	res, err = screstapi.MakeSCRestAPICall(ZCNSCSmartContractAddress, GET_MINT_NONCE, Params{
 		"client_id": client.Id(),
 	})
+
+	return res, err
 }
 
 func GetMiners(active, stakable bool, limit, offset int) ([]byte, error) {
 	if err := CheckConfig(); err != nil {
 		return nil, err
 	}
 
-	return client.MakeSCRestAPICall(MinerSmartContractAddress, GET_MINERSC_MINERS, Params{
+	return node.MakeSCRestAPICallToSharder(MinerSmartContractAddress, GET_MINERSC_MINERS, Params{
```

Comment:
Same here

Reply:
SAA

Reply:
Still we can use "MakeSCRestAPICall" to protect in future. It will not create any problem right?

Reply:
These are not implemented in zbox